dlgr/griduniverse/experiment.py

```diff
@@ -1140,6 +1145,16 @@ def configure(self):
                                                         self.num_participants)
         self.network_factory = self.config.get('network', 'FullyConnected')
 
+        game_config_file = os.path.join(os.getcwd(), GAME_CONFIG_FILE)
```

Comment:
Total nit: we've been moving toward Pathlib, and it seems like we should use it unless there's a reason not to(?)

Reply:
@jessesnyder Sounds good. I trust you to make that update. I'm not sure if I've ever used `Pathlib`